mne_bids/write.py

```diff
@@ -893,23 +899,42 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
                               'overwrite to True.' % bids_fname)
     _mkdir_p(os.path.dirname(bids_fname))
 
-    if verbose:
-        print('Copying data files to %s' % bids_fname)
-
-    if ext not in ALLOWED_EXTENSIONS:
-        raise ValueError('ext must be in %s, got %s'
-                         % (''.join(ALLOWED_EXTENSIONS), ext))
-
     # Copy the imaging data files
-    if ext in ['.fif']:
+    if ext not in ALLOWED_EXTENSIONS[kind]:
+        if kind == 'meg':
+            if verbose:
+                print('Converting data files to Fractal Image Format (FIF)')
```

Comment:
FIF in this case stands for [Functional Image File](http://www.nmr.mgh.harvard.edu/CFNT/sofMNE.html)

Reply:
Oops :)